src/components/layout/inputs/PeriodSelector/PeriodSelector.jsx

```diff
@@ -19,9 +19,14 @@ const PeriodSelector = ({
   periodEndingDate,
   todayDate,
 }) => {
+
+  const focusEndCalendar = () => {
+    const endCalendar = document.getElementsByClassName('endCalendar')[0]
+    endCalendar.click()
+  }
```

Comment:
c'est pas très react comme manière de faire...

```suggestion
  const endDateInput = useRef(null)
  const focusEndDate = () =>  endDateInput.current && endDateInput.current.focus()
```